### PR TITLE
install: Add i2c enabler to make-prep.sh

### PR DESCRIPTION
**Pull request by rzr**
Should work with default config.txt where line is commented.

This is needed for building generic-sensors-addons:

https://discourse.mozilla.org/t/creating-i2c-add-on/26696/4

Change-Id: Ifb8006ded46ce78e537b1a5bf74453fcd635abb8
Signed-off-by: Philippe Coval <philippe.coval@osg.samsung.com>

### CODE REVIEW

**Comment by rzr** [comment]
Relate-to: https://github.com/mozilla-iot/addon-list/pull/54

File: image/make-prep.sh [hunk 1]
```diff
@@ -283,6 +317,7 @@ main() {
     echo "  IMG_FILENAME = ${IMG_FILENAME}"
     echo " PREP_FILENAME = ${PREP_FILENAME}"
     echo "ENABLE_CONSOLE = ${ENABLE_CONSOLE}"
+    echo "ENABLE_I2C = ${ENABLE_I2C}"
```
Comment (dhylands) [inline image/make-prep.sh]: nit: The = are lined up on the other prints

**Comment by rzr** [comment]
If you're ok to enable i2c in default images, that will be great, then I will update patch to add --no-i2c option to disable it.

**Comment by dhylands** [comment]
Yeah - I think that makes sense. It won't go out in the 0.4.0 release, but I don't see any reason why it shouldn't be enabled by default.

**Comment by rzr** [comment]
Hi, I checked on master  (also with  --no-i2c), so it should be safe to merge for v0.4.1

**Pull request merged**